src/components/SignUp/SignUp.css

```diff
@@ -0,0 +1,30 @@
+.textInput {
+    background-color: white;
+    width:145.98px;
+    padding: 6px 11px;
+    font-family: Raleway;
```

Comment:
It doesn't look like this is working. I know that the CSS variables are still being worked on, but I'm going to hold off on merging until we get that up and running.